custom/enikshay/reports/filters.py

```diff
@@ -36,3 +42,58 @@ def filter_context(self):
         context = super(EnikshayLocationFilter, self).filter_context
         context['endpoint'] = self.pagination_source
         return context
+
+
+class QuarterFilter(BaseReportFilter):
```

Comment:
this is needed (in addition to [this one](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/reports_core/filters.py#L132)) because the report is rendered outside the UCR framework? I think it would be good to share more code between these two classes.